docs/publishing-your-site.md

```diff
@@ -91,6 +91,9 @@ Now, when a new commit is pushed to either the `master` or `main` branches,
 the static site is automatically built and deployed. Push your changes to see
 the workflow in action.
 
+After the first successful build, visit `github.com/<username>/<repository>/settings/pages`
```

Comment:
Is this really necessary? When I tested it, it always automatically set the default branch, unless you changed it.

Reply:
Well I just had my first time setup and had to do this setting. I can replay it one a fresh repository, but I assume you want to check yourself before (if) applying this change.

Reply:
I see. Could we maybe reformulate the instructions saying "if the page doesn't show up after a few minutes, check your `gh-pages` setting here...", so maybe more like troubleshooting advice.